xml/virt_support.xml

```diff
@@ -11,6 +11,11 @@
    <dm:bugtracker></dm:bugtracker>
   </dm:docmanager>
  </info>
+ <important>
+  <para>
+   The only supported emulation for all hardware architectures is &qemu;.
```

Comment:
QEMU is supported for full system emulation, and KVM is the only accelerator supported. All other accelerators (TCG, ...) are not supported.